src/pages/Home/index.tsx

```diff
@@ -37,11 +47,24 @@ const Home = () => {
           {vouchers?.data
             ? vouchers.data?.map((voucher) => (
                 <VoucherCard
+                  isRedeemable = {false}
+                  hasRedeemed = {true}
                   voucherID={voucher.voucher_id}
                   onClick={() => openModal(voucher.voucher_id)}
                 />
               ))
             : arr.map(() => <VoucherCardSkeleton />)}
+
+          {dynamicVouchers?.unredeemed
+            ? dynamicVouchers.unredeemed?.map((voucher) => (
+                <VoucherCard
+                  isRedeemable = {true}
+                  hasRedeemed = {false}
```

Comment:
Same nitpick as before :D

Reply:
Another design that you can adopt is to give a default value to `false` for `isRedeemable` and `hasRedeemed`. This way, you do not need to specify that they are `false` when they are, and only need to specify the prop when it is `true`